src/services/worker/worker.service.ts

```diff
@@ -47,19 +47,40 @@ export class WorkerService implements OnModuleInit {
         case SystemType.EVENT_CLUSTER_DEPOSITED:
         case SystemType.EVENT_CLUSTER_WITHDRAWN:
         case SystemType.EVENT_CLUSTER_REACTIVATED:
-        case SystemType.EVENT_OPERATOR_FEE_APPROVED:
         case SystemType.EVENT_VALIDATOR_REMOVED:
           // /Mark as cluster was updated and need to get
           // its fresh metrics in burn-rates task
-          const updated = await this._clusterService.update(
+          (await this._clusterService.update(
             {
               owner: dataItem.owner,
               operatorIds: dataItem.operatorIds,
             },
-            { burnRate: null, cluster: dataItem.cluster },
-          );
-          if (updated) {
+            { burnRate: null, isLiquidated: false, cluster: dataItem.cluster },
+          )) &&
             this._logger.debug(`Updated cluster: ${JSON.stringify(dataItem)}`);
```

Comment:
Can we please avoid this kind of instructions. It is better to assign constant and debug then

Reply:
It makes code less readable and too much specific to js nature, but yes, we cal live with it in case of var scope issue